### DISCUSSION TITLE
Creating Arrow Files with custom gene, genome annotations is failing

### DISCUSSION THREAD

**Discussion by ajitalapati** [GreenleafLab/ArchR#1990] [Questions / Documentation]
I'm trying to create some arrow files with custom gene and genome annotations. They are called BSgenome.Rnorvegicus.UCSC.rn6 and TxDb.Rnorvegicus.UCSC.rn6.refGene, respectively and both able to be downloaded through BiocManager. However, when I continue to make the arrow files for my fragment data, it fails with this message:

Warning message:
In mclapply(..., mc.cores = threads, mc.preschedule = preschedule) :
  4 function calls resulted in an error

I have my cores set to 6, however, even if it's only set to 1, it also doesn't work. If someone knows if this is some versioning error or something like that, please let me know, thanks. I am using the most recent version of R.

**Comment by wangziming-zaza** [comment]
I don't know if your problem has been solved, but I accidentally ran into the same problem when building custom genomes and genome annotation files for bovine. Looking forward to your reply.
